### PR TITLE
Deletebutton

### PR DESCRIPTION
**Pull request by ttaannjjaa**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -17,18 +17,34 @@ function App() {
     navigate('/', { replace: true });
   }
 
+  function handleDelete(id) {
+    const newData = storageData.filter(storeddata => storeddata.id !== id);
+    setStorageData(newData);
+    console.log('ich wurde aufgerufen');
```
Comment (F-Kirchhoff) [inline src/App.js:23]: 🕵️ it is good practice to remove all console.logs when  creating a PR.

File: src/GlobalStyles.js [hunk 2]
```diff
@@ -2,28 +2,29 @@ import { createGlobalStyle } from 'styled-components';
...
 :root {
     --black: #000000;
-    --steelblue: #597A91;  
     --white: #ffffff;
-    --peach: #F4E0D4;
     --grey: #7E6F6F; 
+    --coral: #F88379;
+    --lightblue: #CEE7F6;
+    --peach: #F4E0D4; 
+    --steelblue: #597A91;
```
Comment (F-Kirchhoff) [inline src/GlobalStyles.js:18]: very good that you use variables early on, this saves you a lot of work.

File: src/components/DeleteButton.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components';
+import icon_bin from '../images/icon_bin.svg';
+
+export default function DeleteButton({ handleDelete, id }) {
+  return (
+    <DeleteButtonStyled type="button" onClick={handleDelete} id={id}>
+      <img src={icon_bin} alt="bin icon" width="13px" height="16px" />
```
Comment (F-Kirchhoff) [inline src/components/DeleteButton.js:7]: This is a valid option to  create this button. However you could us a more general button component and pass down the img as a child.

**Review by F-Kirchhoff — APPROVED** [review]
LGTM 🦊

File: src/App.js (hunk 1 above)
Comment (ttaannjjaa) [reply src/App.js:23]: thanks, yes, you are absolutely right. I forgot to remove this

File: src/components/DeleteButton.js (hunk 3 above)
Comment (ttaannjjaa) [reply src/components/DeleteButton.js:7]: thanks for this hint - indeed i was thinking about it, but chose this option as i have not planned another button of this format so far - i will keep in mind to change it, when i should add similar buttons

File: src/components/Navigation.js [hunk 4]
```diff
@@ -30,7 +30,7 @@ const NavigationWrapper = styled.nav`
 const NavButton = styled(NavLink)`
   background-color: var(--peach);
   min-height: 50px;
-  min-width: 33%;
+  min-width: 34%;
```
[the remark this answers is not in the document]
Comment (ttaannjjaa) [reply src/components/Navigation.js:33]: many thanks for pointing this out - a very good reason to refactor the navigation bar code now

File: src/components/Navigation.js [hunk 5]
```diff
@@ -8,33 +8,43 @@ export default function Navigation() {
...
 const NavButton = styled(NavLink)`
   background-color: var(--peach);
   min-height: 50px;
-  min-width: 34%;
   display: flex;
   align-items: center;
   justify-content: center;
   &.active {
-    background-color: ${'var(--steelblue)'};
+    border-top: 5px solid ${'var(--coral)'};
```
Comment (teifler) [inline src/components/Navigation.js:48]: Nice Idea, I will do the same in my code! Looks really clean in general nice, good job!

**Pull request merged**